### PR TITLE
Upgrade flow

### PR DESCRIPTION
**Pull request by Pike**
Adding a WIP PR to publicize my review comments to #1888, and to trigger automated flow-to-ts

### CODE REVIEW

File: frontend/src/modules/entitieslist/components/EntitiesList.js [hunk 1]
```diff
@@ -338,8 +338,6 @@ export class EntitiesListBase extends React.Component<InternalProps> {
         // InfiniteScroll will display information about loading during the request
         const hasMore = props.entities.fetching || props.entities.hasMore;
 
-        console.log('***', props.entities.entities);
-
```
Comment (abowler2) [inline frontend/src/modules/entitieslist/components/EntitiesList.js]: Oops, that was me testing something while helping Mitch.  Thanks for catching it.

File: frontend/src/core/editor/components/FailedChecks.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ export default function FailedChecks(
 }
 
 type MainActionProps = {|
-    source: number | string,
+    source: string,
```
Comment (abowler2) [inline frontend/src/core/editor/components/FailedChecks.js:96]: This was originally typed just as a number and I added the string type based on the use below to fix an error that came up during the upgrade. Which makes me wonder a) why it was of type number to begin with and b) why no errors were thrown before.

Comment (Pike) [reply frontend/src/core/editor/components/FailedChecks.js:96]: There are many things that I tried to break that didn't trigger flow errors. I was pretty underwhelmed by the amount of typesafety in flow.

**Comment by Pike** [comment]
Closing, this is now all in #1888

**Pull request closed without merging**